petastorm/tests/test_parquet_reader.py

```diff
@@ -209,6 +209,33 @@ def preproc_fn1(x):
                (3, 4, 5) == sample['tensor_col_2'].shape[1:]
 
 
+@pytest.mark.parametrize('null_column_dtype', [np.float64, np.unicode_])
+@pytest.mark.parametrize('reader_factory', _D)
+def test_transform_spec_returns_all_none_values(scalar_dataset, null_column_dtype, reader_factory):
```

Comment:
These test cases assume the transform_spec func is creating the null values.  In the more common case, there are  missing values in fields unedited by the transform_spec.  I believe this solution already addresses both cases but it would be good to demonstrate this in the tests either with an additional test case or additional non-edited fields in this test case.